src/Front/Scanner.x

```diff
@@ -348,6 +366,16 @@ gotoStartLine _ _ = do
   alexSetStartCode lineStart
   alexMonadScan
 
+extractMargin :: AlexUserState -> Int
+extractMargin (AlexUserState _ _ _ m) = m
+
+updateMargin :: AlexInput -> Alex ()
```

Comment:
Document this

Reply:
Thank you for the comments, just added documentation and cleaned it up!